### PR TITLE
Add How to add a review

### PR DESCRIPTION
**Pull request by carlvieriste**
Added two files:
* How to add a review
* How to install Ruby (for previewing the site locally)

### CODE REVIEW

File: _posts/2017-01-27-welcome-to-vital-literature-review.markdown [hunk 1]
```diff
@@ -1,6 +1,43 @@
 ---
-layout: post
-title:  "Welcome to the Literature Review site of VITAL."
-date:   2017-01-27 12:18:56 -0500
+layout:    post
+title:     Welcome to the Literature Review site of VITAL
+date:      2017-01-27 12:18:56 -0500
+permalink: /welcome-how-to/
 ---
-On this site, you'll find all the essential papers any deep learning expert should read.
```
Comment (jodoin) [inline _posts/2017-01-27-welcome-to-vital-literature-review.markdown]: replace deep learning expert by 
papers the VITALab members should be aware of.

File: _posts/2017-01-27-welcome-to-vital-literature-review.markdown [hunk 2]
```diff
@@ -1,6 +1,43 @@
...
+## Goal of this website
+
+The goal of this website is to collaborate by helping each other stay up to date with new advances. It will also help people that are new to the field to acquire essential knowledge about the state of the art.
+
+## How to add a review
+
+The process for adding reviews is _git-centric_. Basically, **you just need to add a file to the repo and make a pull request**. Let's go into the details :
+
+1. Clone [the `vitalab.github.io` repo](https://github.com/vitalab/vitalab.github.io) on your computer.
+2. Create the file `YYYY-MM-DD-title-of-your-review.markdown` in the folder `/_posts`. `name-of-your-review` can be the title of the paper, for example. **The format is important, the name of the file must begin by the date of creation of your review.**
```
Comment (jodoin) [inline _posts/2017-01-27-welcome-to-vital-literature-review.markdown]: ... your review.** For exemple, `2017-01-29-You Only Look Once Unified Real-Time Object Detection.markdown` would be a valid name.

File: _posts/2017-01-27-welcome-to-vital-literature-review.markdown [hunk 3]
```diff
@@ -1,6 +1,43 @@
...
+## Previewing your post locally
+
+This site is built around [**Jekyll**](https://jekyllrb.com/). Jekyll takes all the markdown files and generates a static html website.
+
+1.  [Install Ruby using rbenv](/how-to-install-ruby).
+2.  Install Jekyll by running : `gem install bundler jekyll`.
+3.  Go where you cloned the repo and run : `bundle install`. This will install the dependencies for our Jekyll site.
+4.  Run a local webserver using : `bundle exec jekyll serve`.
+
+Note that the site is automatically rebuilt when you make changes to a file.
```
Comment (jodoin) [inline _posts/2017-01-27-welcome-to-vital-literature-review.markdown]: when you change a file

File: how-to-install-ruby.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+~~~
+~~~
+sudo apt-get install git-core curl zlib1g-dev build-essential libssl-dev libreadline-dev libyaml-dev libsqlite3-dev sqlite3 libxml2-dev libxslt1-dev libcurl4-openssl-dev python-software-properties libffi-dev
+~~~
+
+## Get rbenv
+~~~
+cd
+git clone git://github.com/sstephenson/rbenv.git .rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
```
Comment (jodoin) [inline how-to-install-ruby.md]: wouldn't it be better in .profile instead of .bashrc?

Comment (carlvieriste) [inline how-to-install-ruby.md]: My instructions are based on [this tutorial from DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-install-ruby-on-rails-with-rbenv-on-ubuntu-14-04#install-rbenv). See the note in yellow.

I've searched the web ([1](http://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path), [2](http://superuser.com/questions/183870/difference-between-bashrc-and-bash-profile/183980#183980)) and discussed with Clément, and we came to the conclusion that `.bashrc` will be a good option for newbies, and that those who know what they're doing will use `.profile` or `.bash_profile` if they prefer.

File: _posts/2017-01-27-welcome-to-vital-literature-review.markdown (hunk 1 above)
Comment (carlvieriste) [inline _posts/2017-01-27-welcome-to-vital-literature-review.markdown]: I was removing this line, but I re-added it with the change you requested.

**Pull request merged**